templates/poll/notify.tpl

```diff
@@ -1 +1 @@
-<script>Poll.load({ pollid: {pollid} })</script>
```

Comment:
These outside curly braces were being converted to `&#123;` and `&#125` which are the html character codes for the curly braces. I tried escaping them with forward and back slash and everything else i could think of. I eventually settled on refactoring.